index.html

```diff
@@ -50,6 +52,250 @@ <h1>Hey there. I’m Mariane<br />
+                <div class="language-img">
+                    <img src="./images/icon-language.png" alt="language-skill" />
+                </div>
+
+                <div class="skills-name">
+                    <h5>Languages</h5>
+                    <ol class="tech-skill-name">
+                        <li class="lang-skill-name">
+                            <p>JavaScript</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Ruby</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Html</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>CSS</p>
+                        </li>
+                    </ol>
+                </div>
+            </div>
+
+            <div class="programming-skills">
+                <div class="language-img">
+                    <img src="./images/programmin-skills.png" alt="programming" />
+                </div>
+
+                <div class="skills-name">
+                    <h5>Frameworks</h5>
+                    <ol class="tech-skill-name">
+                        <li class="lang-skill-name">
+                            <p>Bootstrap</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Ruby on rails</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>RSpec</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Capybara</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Selenium</p>
+                        </li>
+                    </ol>
+                </div>
+            </div>
+
+            <div class="programming-skills">
+                <div class="language-img">
+                    <img src="./images/skills-icon.png" alt="img" />
+                </div>
+
+                <div class="skills-name">
+                    <h5>Skills</h5>
+                    <ol class="tech-skill-name">
+                        <li class="lang-skill-name">
+                            <p>Codekit</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Github</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Codepen</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Gitlab</p>
+                        </li>
+                        <li class="lang-skill-name">
+                            <p>Terminal</p>
+                        </li>
+                    </ol>
+                </div>
+            </div>
```

Comment:
- [ ] And lastly, please wrap your about me part into one section element. In your code from the line `189` to `273` 🍀😀. If you do all the suggestions I gave you, you should be approved by the next reviewer 😊

Reply:
Well implemented 👍🏽

Reply:
Hi @Rustamxon7 

I have wrapped the about-me into a section.
Thanks for your review 👍